frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -566,23 +567,23 @@ function StartJobForm(props: StartJobFormProps) {
   initialOutputSegmentationLayerName = `${
     initialOutputSegmentationLayerName || "segmentation"
   }_corrected`;
-  // TODO: Other jobs also have an output segmentation layer. The names for these jobs should also be configurable.
-  const hasOutputSegmentationLayer = jobName === "materialize_volume_annotation";
+  const hasOutputSegmentationLayer = jobTypeWithConfigurableLayerName.indexOf(jobName) > -1;
```

Comment:
these two variables names should match better imo. either, use "[output]SegmentationLayer" in both cases or just "outputLayer".

Reply:
I hope this is what you meant :see_no_evil:.